docs/milestone-1.md

```diff
@@ -41,9 +41,23 @@ The acceptance criteria are:
 ### Prerequisites
 
 1. Run Vault (check [How to Run Vault](https://github.com/scality/Vault/blob/development/7.10/TESTING.md#how-to-run-1))
-
+    ```sh
+        $ git clone git@github.com:scality/vault.git
+        $ cd vault
+        $ git checkout development/7.10
+        $ yarn install
+        $ chmod 400 ./tests/utils/keyfile
+        $ VAULT_DB_BACKEND=LEVELDB yarn start #if one wants database to persist
+        $ VAULT_DB_BACKEND=MEMORY yarn start #if one wants database to persist only in process memeory(while the service is running)
+    ```
```

Comment:
If we are specifying the exact steps here. We need to specify the operating system on which these commands will work as well.
and which packages to install.